scripts/quests/otherAreas/X_Marks_The_Spot.lua

```diff
@@ -0,0 +1,131 @@
+-----------------------------------
+-- X Marks the Spot
+-----------------------------------
+-- Log ID: 4, Quest ID: 65
+-----------------------------------
+-- CoP Mission 2-5 Ancient Vows Completed
```

Comment:
You could add the !addmission code for this here to make it easier for people to test